src/auth/signup-form/signup-form.js

```diff
@@ -1,32 +1,158 @@
+                label="First Name"
+                type="Text"
+                fullWidth
+                size="small"
+                name="first_name"
+                value={this.state.first_name}
+                 onChange={this.onChange}
+                variant="outlined"
+              />
+            </Grid>
+            <Grid item xs={6}>
+              <TextField
+                id="outlined-primary"
+                label="Last Name"
+                type="Text"
+                fullWidth
+                size="small"
+                name="last_name"
+                value={this.state.last_name}
+                 onChange={this.onChange}
+                variant="outlined"
+              />
+            </Grid>
+            <Grid item xs={12}>
+              <TextField
+                id="outlined-primary"
+                label="Email"
+                type="email"
+                fullWidth
+                size="small"
+                name="email"
+                value={this.state.email}
+                 onChange={this.onChange}
+                variant="outlined"
+              />
+            </Grid>
+            <Grid item xs={12}>
+              <TextField
+                id="outlined-primary"
+                label="Phone Number"
+                type="Number"
+                fullWidth
+                size="small"
+                name="phone_no"
+                value={this.state.phone_no}
+                 onChange={this.onChange}
+                variant="outlined"
+              />
+            </Grid>
+            <Grid item xs={12}>
+            <TextField
+              id="outlined-primary"
+              label="Description"
+              type="Text"
+              fullWidth
+              size="small"
+              name="description"
+              value={this.state.description}
+               onChange={this.onChange}
+              variant="outlined"
+            />
+          </Grid>
+            <Grid item xs={12}>
+              <TextField
+                id="outlined-primary"
+                label="Password"
+                type="password"
+                fullWidth
+                size="small"
+                name="password"
+                value={this.state.password}
+                 onChange={this.onChange}
+                variant="outlined"
+              />
+            </Grid>
+            <Grid item xs={12}>
+              <TextField
+                id="outlined-primary"
+                label="Confirm Password"
+                type="password"
+                fullWidth
+                size="small"
+                name="cnf_password"
+                value={this.state.cnf_password}
+                 onChange={this.onChange}
+                variant="outlined"
+              />
+            </Grid>
+            <Button
+              className="savebtn"
+              type="submit"
+              variant="contained"
+              color="primary"
+            >
```

Comment:
my bad sir, making changes but cannot extract the css completely because it also contain the styling for positioning and padding.So, removing the unnecessary css content

Reply:
@vaibhavdaren Just made the required changes Please review